hs/src/Reach/EditorInfo.hs

```diff
@@ -70,6 +62,22 @@ instance Show CompletionItemKind where
 completionKind :: SLVal -> Maybe CompletionItemKind
 completionKind v =
   case v of
+    SLV_Null _ _ -> Just CK_Constant
+    SLV_Bool _ _ -> Just CK_Constant
+    SLV_Int _ _ -> Just CK_Constant
+    SLV_Bytes _ _ -> Just CK_Constant
+    SLV_Array _ _ _ -> Just CK_Constant
+    SLV_Tuple _ _ -> Just CK_Constant
+    SLV_Object _ _ _ -> Just CK_Constant
+    SLV_Struct _ _ -> Just CK_Constant
+    SLV_Clo _ _ _ -> Just CK_Function
+    SLV_Data _ _ _ _ -> Just CK_Constructor -- ?
+    SLV_DLC _ -> Just CK_Constant
+    SLV_Connector _ -> Nothing
+    SLV_RaceParticipant _ _ -> Just CK_Constant -- ?
+    SLV_Participant _ _ _ _ -> Just CK_Constant -- ?
+    SLV_Map _ -> Just CK_Variable -- ?
+    SLV_Deprecated _ _ -> Nothing -- ?
```

Comment:
You could go inside and show what the thing inside is or leave it this way and they won't show